### PR TITLE
Add Card component

### PR DESCRIPTION
**Pull request by galloween**

### CODE REVIEW

File: projects/ui-framework/src/lib/cards/card-add/card-add.component.scss [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+@import "../../style/mixins.scss";
+
+:host {
+
+  background-color: $grey-200;
+  border-style: dashed;
+  user-select: none;
+  cursor: pointer;
+
+  &, &:hover {
```
Comment (omrihecht) [inline projects/ui-framework/src/lib/cards/card-add/card-add.component.scss]: the first & is empty, i think you can remove

File: projects/ui-framework/src/public_api.ts [hunk 2]
```diff
@@ -150,6 +150,9 @@ export * from './lib/table/table/table.interface';
 // Single card
 export { CardModule } from './lib/cards/card/card.module';
 export { CardComponent } from './lib/cards/card/card.component';
+// Add new Card
+export { CardAddModule } from './lib/cards/card-add/card-add.module';
```
Comment (omrihecht) [inline projects/ui-framework/src/public_api.ts]: I think the module card can export cardComponent and cardAddComponent

File: projects/ui-framework/src/lib/style/shapes.scss [hunk 3]
```diff
@@ -63,3 +63,10 @@ $chevronWidth: 2px;
   justify-content: center;
   align-items: center;
 }
+
+@mixin cross($color, $size, $gauge) {
+  background-image: linear-gradient($color,$color),linear-gradient($color,$color);
+  background-size: $size $gauge, $gauge $size;
```
Comment (omrihecht) [inline projects/ui-framework/src/lib/style/shapes.scss]: $gauge = line width ?

File: projects/ui-framework/src/lib/cards/card-add/card-add.component.scss (hunk 1 above)
Comment (galloween) [reply projects/ui-framework/src/lib/cards/card-add/card-add.component.scss]: no, the first is host

File: projects/ui-framework/src/lib/style/shapes.scss (hunk 3 above)
Comment (galloween) [reply projects/ui-framework/src/lib/style/shapes.scss]: yes ;)

File: projects/ui-framework/src/lib/cards/card-add/card-add.component.scss (hunk 1 above)
Comment (galloween) [reply projects/ui-framework/src/lib/cards/card-add/card-add.component.scss]: image

**Pull request merged**